mantidproject/mantid
# Set the phi value to 0 in XYE MAUD headers

**Description of work.**
This PR changes the Phi value saved in MAUD headers to 0

**To test:**
- Run the following:

```
CreateSampleWorkspace(OutputWorkspace='123', NumBanks=1, BankPixelWidth=1, XUnit='DSpacing')
SaveFocusedXYE(InputWorkspace='123', Filename=Path/To/Outfile.ascii, Format='MAUD')
```

- Check the resulting file has Phi set to 0 (first few lines)

Fixes #23154 . 

Does this update require release notes?

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
